content/kubermatic/master/tutorials_howtos/project_and_cluster_management/cluster_defaulting/_index.en.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Cluster defaulting"
```

Comment:
For future I'd recommend to rename this to "Cluster Template" or something similar